src/vaadin-date-time-picker.html

```diff
@@ -355,6 +389,11 @@
 
         ready() {
           super.ready();
+
+          if (this.checkValidity.toString() !== customElements.get('vaadin-date-time-picker').prototype.checkValidity.toString()) {
+            this.$.customField.checkValidity = () => this.checkValidity();
+          }
```

Comment:
What is the purpose of this? It's a bit confusing to me. Also all tests pass even if this is removed so it doesn't seem to be needed or it's not covered by tests.

Reply:
Didn't come up with the tests in the beginning, but now have added those.

The idea is that validation in `vaadin-dtp` is based on validation in the `custom-field`, so in order it to be called at the right point of time I need to propagate it to the internal `customField`.

Arrow function is used in order to provide the proper scope of the execution (`this === vaadin-date-time-picker`).